axelrod/tests/unit/test_backstabber.py

```diff
@@ -47,3 +47,51 @@ def test_reset(self):
         P1.reset()
         self.assertEqual(P1.D_count, 0)
         self.assertEqual(P1.history, [])
+
+
+class DoubleCrosser(TestPlayer):
+
+    name = "DoubleCrosser"
+    player = axelrod.DoubleCrosser
+    stochastic = False
+
+    def test_initial(self):
+        P1 = axelrod.DoubleCrosser()
+        P2 = axelrod.Player()
+        self.assertEqual(P1.strategy(P2), P1._initial)
+
+    def test_strategy(self):
+        """
+        Forgives the first 3 defections but on the fourth
+        will defect forever. If the opponent did not defect
```

Comment:
Yes, I've made a bit of a mistake there. I'll fix that for both backstabber and doublecrosser.